File: src/main/java/project/rpg/annotation/skill.java
package project.rpg.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Target;

@Target({ElementType.METHOD, ElementType.TYPE})
public @interface skill {  //심심해서 추가해봄 마커 어노테이션임
    String name();
}

File: src/main/kotlin/project/rpg/annotation/skill.kt
package project.rpg.annotation

@Target(
    AnnotationTarget.FUNCTION,
    AnnotationTarget.PROPERTY_GETTER,
    AnnotationTarget.PROPERTY_SETTER,
    AnnotationTarget.CLASS
)
annotation class skill( //심심해서 추가해봄 마커 어노테이션임
    val name: String
)
